docs/rfc/active/tags/README.md

```diff
@@ -0,0 +1,210 @@
+   * Audit stamp associated with employment of this tag to this entity
+   */
+   applicationStamp: AuditStamp
+}
+```
+
+### `Tags` container entity
+
+Then we define a `Tags`-model, which is used as a container for tag employments. This container is taken into use in
+aspects.
+
+```
+namespace com.linkedin.common
+
+/**
+ * Tags information
+ */
+record Tags {
+
+   /**
+   * List of tag employments
+   */
+   elements: array[TagAttachment] = [ ]
+}
+```
+
+## How we teach this
+
+We should create/update user guides to educate users for:
+
+- Suggestions on how to use tags: low threshold metadata-addition, and the possibility of doing new types of searches
+
+## Drawbacks
+
+This is definitely more complex than just adding strings to an array.
+
+## Alternatives
+
+An array of string is a simple solution but does allow for the same functionality as suggested here.
+
+Apache Atlas uses a similar approach. The require you to create a Tag instance before it can be associated with an
+"asset", and the attachment is done using a dropdown list. The tags can also have attributes and a description. See
+[here](https://docs.cloudera.com/HDPDocuments/HDP2/HDP-2.5.3/bk_data-governance/content/ch_working_with_atlas_tags.html)
+for an example. The tags are a central piece in the UI and readably searchable, as easily as datasets.
+
+Atlas also has concept very closely related to tags, called _classification_. Classifications are similar to tags in
+that they need to be created separately, can have attributes (but no description?) and are attached to assets is done
+using a dropdown list. Classifications have the added functionality of propagation, which means that they are
+automatically applied to downstream assets, unless specifically set to not do so. Any change to a classification (say an
+attribute change) also flows downstream, and in downstream assets you're able to see from where the classification
+propagated from. See
+[here](https://docs.cloudera.com/HDPDocuments/HDP3/HDP-3.1.5/using-atlas/content/propagate_classifications_to_derived_entities.html)
+for an example.
+
+## Rollout / Adoption Strategy
+
+Using the functionality is optional and does not break other functionality as is. The solution is generic enough that
+the users can easily take into use. It can be take into use as any other entity and aspect.
+
+## Future Work
+
+- add `Tags` to aspects for entities.
+- Implement relationship builders as needed.
+- The implementation of and need for access control to tags is an open question
+- As this is first and foremost a tool for discovery, the UI work is extensible:
+  - Creating tags in a way that makes duplication and spelling mistakes difficult.
+  - Attaching tags to entities: autocomplete, dropdown, etc.
+  - Visualizing existing tags, and which are most popular?
+- Explore the concept of tagged tags for building tag hierarchies. What are the possible uses case (for example
+  "sensitivity" and "PII")? How is this visualized in the UI?
```

Comment:
I don't think "tag" (especially it's a tag on aspect level) is the best solution for regulation compliance. LinkedIn team has several mature solution and experience which we believe can share and benefit open source world.

Reply:
Ok, this sensitivity solution is clearly a red herring, so I'll change it 😄

Reply:
That being said, open sourcing your sensitivity implementation would be greatly appreciated